decidim-conferences/spec/commands/create_conference_spec.rb

```diff
@@ -173,6 +173,31 @@ module Decidim::Conferences
           expect(linked_processes.first).to eq(process_two)
         end
       end
+
+      context "when linking unpublished linked_participatory_space_resources" do
+        let!(:process_one) { create :participatory_process, :unpublished, organization:, weight: 2 }
+        let!(:process_two) { create :participatory_process, organization:, weight: 1 }
+        let(:related_process_ids) { [process_one.id, process_two.id] }
+        let!(:consultation_one) { create :consultation, :unpublished, organization: }
+        let!(:consultation_two) { create :consultation, organization: }
+        let(:related_consultation_ids) { [consultation_one.id, consultation_two.id] }
+
+        it "sorts by created at" do
```

Comment:
You have copy-pasted this "it" section and you forgot to change the text inside it 😛